tests/telemetry/test_telemetry.py

```diff
@@ -122,13 +140,14 @@ def setup_streaming_telemetry(duthosts, rand_one_dut_hostname, localhost,  ptfho
     file_exists = ptfhost.stat(path=gnxi_path + "gnmi_cli_py/py_gnmicli.py")
     pytest_assert(file_exists["stat"]["exists"] is True)
 
+
 def skip_201911_and_older(duthost):
     """ Skip the current test if the DUT version is 201911 or older.
     """
     if parse_version(duthost.kernel_version) <= parse_version('4.9.0'):
```

Comment:
>kernel_version [](http://example.com/codeflow?start=29&length=14)

You can check similar skip function in the codebase, no need to check kernel version.

Reply:
Please be aware this is not the new change introduced by this PR.